modules/end-user-guide/pages/mounting-git-configuration.adoc

```diff
@@ -46,6 +46,11 @@ __<ConfigMap_prepared_in_step_1>__
 EOF
 ----
 
+[NOTE]
```

Comment:
Let's put the note at the top of the procedure, before pre-requirements

Reply:
I've changed the configmap block to not to specify just username and email but generalise the gitconfig content:

![screenshot-0 0 0 0_4000-2024 02 22-16_31_27](https://github.com/eclipse-che/che-docs/assets/7668752/5ac6706d-8b48-4c6a-b6b0-929df8ec15c7)

 With this change I think the note suits here more as it specifies that the general gitconfig content can be updated automatically.

Reply:
@vinokurig this NOTE should be placed at the very beginning, just after the title `Mounting Git configuration`

Reply:
@ibuziuk done